x-pack/plugins/security_solution/cypress/tasks/timeline.ts

```diff
@@ -79,7 +73,6 @@ export const openTimelineSettings = () => {
 };
 
 export const populateTimeline = () => {
-  executeTimelineKQL(hostExistsQuery);
```

Comment:
what are you removing this?

Reply:
we populate results into the timeline by default, so we don't need to that manually